components/finances-section/association/index.tsx

```diff
@@ -17,10 +17,16 @@ const ColorCircle = ({ color }: { color: string }) => (
 const colorResultat = constants.chartColors[1];
 const colorCA = constants.chartColors[4];
 
+/**
+ * We use to have finances for association but data disappeared from open data API.
+ *
+ * @param param0
+ * @returns
+ */
 export const FinancesAssociationSection: React.FC<{
-  association: IAssociation;
+  association: IDataAssociation;
```

Comment:
A small interrogation on naming. Depending on where you are in the codebase, `association` variable can either be `uniteLegale` that has been type checked to be association, or `AssociationData`. Should we unify name and type and systematically name `IDataAssociation` variable `associationData`? 

Or use `uniteLegale` systematically for `IAssociation`?

I favor the second option, but it's not blocking for this PR, can be done incrementally.

Reply:
I also favour the second option. Will make the change azap